models/mart/mrt_monthly_order_report.sql

```diff
@@ -0,0 +1,43 @@
+WITH monthly_users_recap AS (
+
+
+SELECT DATE_TRUNC(order_date,month) AS order_month,
+COUNT(DISTINCT user_name) AS total_monthly_users
+FROM {{ source('sales_database', 'order')}}
+GROUP BY 1
+ORDER BY total_monthly_users DESC
+
+
+), total_monthly_user_from_jawa_timur AS (
+SELECT DATE_TRUNC(order_date,month) AS order_month,
+COUNT(DISTINCT user.user_name) AS total_monthly_users_from_jawa_timur
+FROM {{ source('sales_database', 'order')}} AS orders
+LEFT JOIN {{ source('sales_database', 'user')}} AS user ON user.user_name = orders.user_name
+WHERE user.customer_state LIKE '%JAWA%TIMUR%'
+GROUP BY order_month
+
+
+), monthly_orders_recap AS (
+
+
+SELECT DATE_TRUNC(order_date,month) AS order_months,
+COUNT(order_id) AS total_monthly_orders
+FROM {{ source('sales_database', 'order')}}
+GROUP BY order_month
+
+
+), shipping_cost AS (
+
+
+SELECT shipping_cost
+FROM sales_database.order_item
+WHERE price > 7000
+)
```

Comment:
```suggestion
)
```
This subquery isn't used in the final query, you can delete it